### ISSUE TITLE
cannot enter toolbox from zsh shell

### ISSUE THREAD

**Issue by mvyletel** [debarshiray/fedora-toolbox#13]
simply because the base image does not contain zsh. installing it fixes the problem of course

```[mv@lenivo]~% fedora-toolbox --release 30 -v enter
exec failed: container_linux.go:337: starting container process caused "exec: \"/usr/bin/zsh\":
stat /usr/bin/zsh: no such file or directory"
exit status 1
```
another small issue is that zsh does not expect PS1 but this not much of a problem

**Comment by mvyletel** [comment]
one way to fix this would be to make sure that $SHELL is present in toolbox. if not fallback to bash. or even better install $SHELL in toolbox

**Comment by debarshiray** [comment]
The fallback to Bash if `$SHELL` is unavailable is already addressed by commit 912f3000a9e4a43f and commit 48ddba630642507f in Git master. Sadly, those are not part of the 0.0.1 release, which is why you are still suffering from the problem.

**Comment by mvyletel** [comment]
oh, should have checked the code first :) closing this. thanks
